### DISCUSSION TITLE
How to layout like this?

### DISCUSSION THREAD

**Discussion by junmaqiang** [fyne-io/fyne#3857] [Q&A] [answered]
There must be three columns, the left and right columns have the same width, and the rest of the width is for the middle column, which can be adaptive

Here is how I write it now
```
	// 
	lefts := canvas.NewRectangle(color.NRGBA{0x62, 0x76, 0x99, 0xff})
	lefts.SetMinSize(fyne.NewSize(260, 600))
	// 
	conts := canvas.NewRectangle(color.NRGBA{0x11, 0x22, 0x33, 0xff})
	conts.SetMinSize(fyne.NewSize(360, 600))
	// 
	rights := canvas.NewRectangle(color.NRGBA{0x62, 0x76, 0x88, 0xff})
	rights.SetMinSize(fyne.NewSize(260, 600))
	// 
	con := container.NewAdaptiveGrid(3, lefts, conts, rights)
```

**Comment by andydotxyz** [comment, marked as answer]
I think the Border container will do what you want, though I would be very careful about setting a large min-size for items as it will probably break smaller screens.
